### PR TITLE
First draft of Code of Conduct

### PR DESCRIPTION
**Pull request by Bhupesh-V**
The COC for both Reddit and Github.
this will also apply to any future projects, events etc

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+In order to keep the DevelopersIndia community warm, welcoming, and safe place we ask that all participants read and adhere to the following code of conduct.
```
Comment (Jarmos-san) [inline CODE_OF_CONDUCT.md:1]: .....and a safe place, we ask....

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Rules
+* Be kind and courteous to your fellow members.
+* Criticize ideas, not people.
+* Discuss publicly. Refrain from direct messaging people unless it is specifically requested.
```
Comment (Jarmos-san) [inline CODE_OF_CONDUCT.md:6]: Discuss publicly & refrain.... (omit the period & insert "_and_")

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+* Don't be offensive
+* Ask questions; don't ask to ask. There’s no need to write "Is there anyone that can help me with X?" Just ask about X.
+# Prohibitions
```
Comment (Jarmos-san) [inline CODE_OF_CONDUCT.md:9]: Shouldn't "_Prohibitions_" be preceded by a newline?

**Review by Jarmos-san — COMMENTED** [review]
Looks good enough for starters except a few nit picks from my end 😆 Can merge is Sid approves it.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (Bhupesh-V) [reply CODE_OF_CONDUCT.md]: done

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (Bhupesh-V) [reply CODE_OF_CONDUCT.md]: done

**Review by Jarmos-san — APPROVED** [review]

**Pull request merged**